.circleci/config.yml

```diff
@@ -86,3 +103,9 @@ workflows:
           filters:
             branches:
               only: master
+      - upload_artifacts:
+          filters:
+            branches:
+              only: master
```

Comment:
@XVincentX If we're only doing this on master, will `${CIRCLE_TAG}` have a value? Will ghr know what version number this is?

Reply:
Oh this is a very good point — I think for the way semantic release works the tag is never tested on CircleCI, we're going to need to find another way around it. I think we can get away by doing a simple `git tag` and grab the latest one since the release has happened already.

Reply:
Latest is no good we might release v4.0 then 43.2.1